docs/source/introduction/platform-install.rst

```diff
@@ -246,10 +247,13 @@ to connect to empd (port 4369) on <hostname>."
 
 Step 4 - Bootstrap the environment
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Remove older version of rabbitmq_server directory. Defaults to <user_home>/rabbitmq_server/rabbitmq_server-3.9.7
```

Comment:
Why am I reading this for a second time?

Reply:
this is part of both readme.md and platform-install.rst